Automattic/vip-go-mu-plugins
# Added before_invoke hook for wp media regenerate.

## Description
Adds before_invoke hook for media regenerate command.

If the --skip-delete flag is omitted from the media regenerate command, the before_invoke hook function will recognize this and add it to the CLI arguments, and re-run the command.

Messaging will be displayed to the user to inform them that this is happening.

![Screen Shot 2021-12-16 at 3 20 32 PM](https://user-images.githubusercontent.com/46167019/146457875-69d2234f-1df0-4cd9-aeef-bb2bf25b7095.png)

## Changelog Description

A Change has been made which will force the us of the "--skip-delete" flag when the WP CLI "media regenerate" command is run. This will prevent the media regenerate command from inadvertently deleting original images in the VIP file system.

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test
1. Check out PR.
1. Run the command `wp media regenerate`
1. verify that the command fails to execute with error
